From 07d3ceabd39d70496cd2f22f9800948ec73098c7 Mon Sep 17 00:00:00 2001
From: Tatsuya Matsuo <96454784+matsuo-tat@users.noreply.github.com>
Date: Fri, 15 Dec 2023 17:33:58 +0900
Subject: [PATCH 03/11] [xcvrd] Set host_tx_ready default to true

This change sets host_tx_ready default to true. This is for environments
that do not support the CONFIG_DB PORT table host_tx_ready field
configuration.
---
 sonic-xcvrd/xcvrd/xcvrd.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sonic-xcvrd/xcvrd/xcvrd.py b/sonic-xcvrd/xcvrd/xcvrd.py
index d685890..ec3c14a 100644
--- a/sonic-xcvrd/xcvrd/xcvrd.py
+++ b/sonic-xcvrd/xcvrd/xcvrd.py
@@ -1313,7 +1313,7 @@ class CmisManagerTask(threading.Thread):
         return float(power)
 
     def get_host_tx_status(self, lport):
-        host_tx_ready = 'false'
+        host_tx_ready = 'true'
 
         asic_index = self.port_mapping.get_asic_id_for_logical_port(lport)
         state_port_tbl = self.xcvr_table_helper.get_state_port_tbl(asic_index)
-- 
2.20.1

